packages/schematics/angular/ng-new/index_spec.ts

```diff
@@ -103,4 +103,22 @@ describe('Ng New Schematic', () => {
     const { cli } = JSON.parse(tree.readContent('/bar/angular.json'));
     expect(cli.packageManager).toBe('npm');
   });
+
```

Comment:
These tests are more appropriately placed in the application schematic spec file, as they test code within the application schematic. None of this code tested code is in `ng-new`.

Reply:
Hi @alan-agius4, do you suggest me to move the test cases inside application? In the index.spec file?

Reply:
Yes